modules/shared/src/main/scala/io/constellationnetwork/env/JarSignature.scala

```diff
@@ -31,6 +38,9 @@ object JarSignature {
     }
   }
 
+  def readAllBytesFromDir[F[_]: Files](dir: Path): Stream[F, Byte] =
+    Files[F].walk(dir).evalFilter(Files[F].isRegularFile).filter(p => p.endsWith("semanticdb")).flatMap(Files[F].readAll)
```

Comment:
you want to calculate based on semanticdb files only, or you wanted to exclude it?

Reply:
:facepalm: that always trip me :P